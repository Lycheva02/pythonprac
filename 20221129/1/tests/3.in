class near_dot(metaclass=dump):
    def __init__(self, dot, eps):
        self.dot = dot
        self.eps = eps
    def near(self, andot):
        return (abs(andot - self.dot) <= self.eps)

a = near_dot(10, 0.03)
print(a.near(16))
print(a.near(9.996))