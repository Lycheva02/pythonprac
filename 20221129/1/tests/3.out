__init__: (10, 0.03), {}
near: (16,), {}
False
near: (9.996,), {}
True
